.markdownlint.json

```diff
@@ -6,5 +6,27 @@
   "no-inline-html": false,
   "no-trailing-punctuation": false,
   "no-trailing-spaces": false,
-  "no-multiple-blanks": false
+  "no-multiple-blanks": false,
+  "MD001": false,
+  "MD003": false,
+  "MD004": false,
+  "MD005": false,
+  "MD007": false,
+  "MD010": false,
+  "MD011": false,
+  "MD018": false,
+  "MD022": false,
+  "MD025": false,
+  "MD028": false,
+  "MD030": false,
+  "MD031": false,
+  "MD032": false,
+  "MD034": false,
+  "MD038": false,
+  "MD039": false,
+  "MD040": false,
+  "MD042": false,
+  "MD045": false,
+  "MD046": false,
+  "MD047": false,
```

Comment:
The Markdown files that had been created _prior_ to the bash script that checks for changed files fail these rules.